genai-perf/tests/test_input_image_retrieval.py

```diff
@@ -1,155 +0,0 @@
-# Copyright 2024, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
```

Comment:
why did this whole file get deleted?

Reply:
From the PR description above,
> create unit tests for the converter class that handles chat, vision, and image retrieval endpoints.

Basically ported to/under `test_openai_chat_converter.py`

Reply:
oof. shouldn't have missed that.
thank you